src/core/js/adapt.js

```diff
@@ -227,6 +227,39 @@ define([
 
     };
 
+    // Relative strings describe the number and type of hops in the model hierarchy
+    // "@component +1" means to move one component forward.
+    // This function would return the following:
+    // {
+    //       type: "component",
+    //       offset: 1
+    // }
```

Comment:
maybe just add that this is used by plugins like Trickle.. (any others?). aside from that it's all good.

Reply:
done

Reply:
no others as yet